features/page-visibility.yml

```diff
@@ -0,0 +1,10 @@
+name: Page visibility state
```

Comment:
might want to preclude a future battle for that filename when we get to `api.Document.visibilityState`, known as the Page Visibility API.